code/modules/mob/living/carbon/human/species_types/snail.dm

```diff
@@ -154,7 +156,7 @@
 	)
 	//SKYRAT EDIT END - Roundstart Snails
 
-/datum/armor/backpack_snail
+/datum/armor/backpack_snail //SKYRAT COMMENT CLARIFICATION - Roundstart Snails - These armor values don't actually do any protection of the wearer, this is for checking direct damage to the backpack. Damage resistance stuff is in their heart file.
```

Comment:
Is that armor datum actually something that we added? Because if so, it should be in a modular file, not here.

Reply:
naw that was there to begin with